client/app/components/onboardingGuide/GuideStatusPage.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import scss from './styles.scss';
+
+const GuideStatusPage = (props) => {
+
```

Comment:
No PropTypes for this component? Is it even possible to add those to a stateless function component? If not, I'd vote for using classes everywhere with proper prop type annotations.